lib/src/main/java/com/squareup/cycler/Recycler.kt

```diff
@@ -180,8 +177,8 @@ class Recycler<I : Any> internal constructor(
       // This might not suspend if there's no item comparator and a general refresh is done.
       // That's why we pass the backgroundContext for [generateDataChangesLambdas] to decide.
       val notifications = newUpdate.generateDataChangesLambdas(
-          adapter.itemComparator,
-          backgroundContext
+        adapter.itemComparator,
+        backgroundContext
```

Comment:
The latest version of ktlint broke continuation indent support, so this violates Square's code style (if your IDE is configured correctly, it should undo these unindents). I think you need to use an older version of the plugin (workflow is using 9.2.1), and then configure continuation indent using `continuation_indent_size=4` in .editorconfig.